website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Multi-Tenancy Best Practices"
+linkTitle: "Multi-Tenancy"
+weight: 1
+description: This guide describes the best practices concepts to write Operators for Multi-Tenancy solutions.
+---
+
+## Operator Behavioral Guidelines
+
+### Create resource requests
+
+* for your Operator's own `Pods`
+* for `Pods` your operator creates / manages
+
+The reasoning behind this is, to allow [ResourceQuota][k8s-resources-quotas] configuration
+to work. This is something that every production cluster will follow as a best practice, to prevents tenants
+from being able to consume all resources on the cluster and block other worklaods from being scheduled. 
+[ResourceQuota][k8s-resources-quotas] in a for CPU and RAM to be rejected. For further information see:
+[Requests and limits][k8s-requests-limits]
```

Comment:
this tells me why we should be creating resource requests, but it doesn't seem to tell me HOW.

Reply:
> The reasoning behind this is, to allow [ResourceQuota][k8s-resources-quotas] configuration

The reasoning behind WHAT? I'm confused what "this" refers to?

> to work. This is something that every production cluster will follow as a best practice, to prevents tenants
> from being able to consume all resources on the cluster and block other worklaods from being scheduled. 

I tried to reword this a bit, it would be better stated as follows:

_In order to prevent tenants from consuming all the resources on a cluster or blocking other workloads from being scheduled, many production clusters will DO WHAT?_

But I don't know *what* the cluster will do to prevent that. It wasn't clear from your original paragraph what that is.

> [ResourceQuota][k8s-resources-quotas] in a for CPU and RAM to be rejected. For further information see:
> [Requests and limits][k8s-requests-limits]

ResourceQuote in a for CPU doesn't make sense. I'm not sure what you're trying to say here.